ipn/ipnlocal/local.go

```diff
@@ -403,6 +404,12 @@ type LocalBackend struct {
 	// notified about.
 	lastNotifiedDriveShares *views.SliceView[*drive.Share, drive.ShareView]
 
+	// lastKnownHardwareAddrs is a list of the previous known hardware addrs.
+	// Previously known hwaddrs are kept to work around an issue on Windows
+	// where all addresses might disappear.
+	// http://go/corp/25168
+	lastKnownHardwareAddrs syncs.AtomicValue[[]string]
```

Comment:
You could alternatively use a package level var in the `posture`, since even if we have multiple clients in the same process, posture identity is going to be the same for them.

Reply:
discussed offline, decided against.